atlas

```diff
@@ -289,6 +281,13 @@ display_pull_params() {
 }
 
 pull_translations() {
+  if [ -z "$pull_directory" ];
```

Comment:
Now, at least one directory is needed. Copying the root directory usually means copying a ton of `.gitignore` and other meta-files.

This is a good-to-have change.